src/main/java/uk/gov/mca/beacons/service/model/BeaconPersonRepository.java

```diff
@@ -2,8 +2,8 @@
 
 import java.util.UUID;
 import org.springframework.data.repository.CrudRepository;
-import org.springframework.stereotype.Repository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-@Repository
+@RepositoryRestResource(exported = false)
 public interface BeaconPersonRepository
   extends CrudRepository<BeaconPerson, UUID> {}
```

Comment:
BeaconPerson is the join table for Beacon and Person right? In which case I think we would want to expose the end point, because that's how we do things like "show me all of my beacons"?